src/AssimpLoader.cpp

```diff
@@ -1329,7 +1340,8 @@ bool AssimpLoader::createSubMesh(const Ogre::String& name, int index, const aiNo
             vect.y = norm->y;
             vect.z = norm->z;
 
-            vect *= aiM;
+            vect *= normalMatrix;
```

Comment:
you have to use the inverse transpose of this matrix to account for non uniform scaling

Reply:
done